packages/core-components/src/components/table/table.tsx

```diff
@@ -37,6 +40,13 @@ export class TableComponent {
     }
   }
 
+  private setHeaderResize() {
+    if (this.size === TableSizes.EQUAL) {
+      const allHeaders = this.host.querySelectorAll('b2b-table-header');
```

Comment:
the last header shouldn't be resizable as it automatically grows when the previous element sibling is resized. Currently, trying to resize the last header results in an error due to `offsetWidth` being undefined